src/rgw/rgw_cache.cc

```diff
@@ -106,7 +104,7 @@ int ObjectCache::get(const string& name, ObjectCacheInfo& info, uint32_t mask, r
 bool ObjectCache::chain_cache_entry(std::initializer_list<rgw_cache_entry_info*> cache_info_entries,
 				    RGWChainedCache::Entry *chained_entry)
 {
-  std::unique_lock l{lock};
+  RWLock::WLocker l(lock);
```

Comment:
why would we prefer these legacy Ceph lock types over native c++ types?  I do not

Reply:
apologies for the regresion, RWLock::* replaced later on with std::unique_lock